src/Analyzers/CSharp/Tests/UsePatternMatching/CSharpAsAndNullCheckTests_FixAllTests.cs

```diff
@@ -160,22 +160,22 @@ class Symbol
 
                 void M(object o, bool b0, bool b1)
```

Comment:
this test was affected because the line above is `public ContainingSymbol { get; }`.  Instead of going off the rails, we now properly parse and recover.  So formatting works properly below.